Clinical-Genomics/cg
# Adds delivery date to delivery report

This PR fixes the missing delivery-date on the delivery-reports.

How to test:
0. install on stage on rasta
0. run cg/scripts/add-delivery_report_created_at-to-analysis.sql on cg-stage database
1. create a crontab job that calls `cg upload delivery-reports` 
1. wait until the jobs should have been run
1. open the delivery report of a case in scout

Expected outcome:
The case has a date for delivery report
